stl/inc/chrono

```diff
@@ -5498,26 +5499,23 @@ struct formatter<_CHRONO day, _CharT> {
         } else {
             _Stream.imbue(_FormatCtx.locale());
             tm _Time = {.tm_mday = static_cast<int>(static_cast<unsigned int>(_Val))};
-            for (auto _Spec_it = _Specs._Chrono_specs_list.begin(); _Spec_it != _Specs._Chrono_specs_list.end();) {
+            for (auto _Spec_it = _Specs._Chrono_specs_list.begin(); _Spec_it != _Specs._Chrono_specs_list.end();
+                 ++_Spec_it) {

```

Comment:
Suggestion (sorry for not seeing this earlier): This can now be simplified all the way to a read-only range-for: `for (const auto& _Spec : _Specs._Chrono_specs_list)` since we don't need to advance the iterator manually etc.